### ISSUE TITLE
API not reading block author.

### ISSUE THREAD

**Issue by bwhm** [Joystream/apps#136]
Using the apps API, it is currently not possible to retrieve the block author. It works with on polkadot. How to reproduce:
Take code snippet below
```
let count = 0;
const unsub = await api.derive.chain.subscribeNewHead((header) => {
  console.log(`#${header.blockNumber}:`, header.author);

  if (++count === 5) {
    console.log('5 headers retrieved, unsubscribing');
    unsub();
  }
});
```
paste it in [polkadot-js](https://polkadot.js.org/apps/#/js)
and in [Pioneer](https://testnet.joystream.org/athens/pioneer/#/js).

Note for myself and others: The `controller` is getting credit for signing, NOT the `session` key.

**Comment by mnaamani** [comment]
I'll take a look.

**Comment by mnaamani** [comment]
Yes confirmed this is resolved by getting latest updates from upstream in #137 
<img width="1157" alt="Screen Shot 2019-04-19 at 12 54 56 AM" src="https://user-images.githubusercontent.com/1621012/56393668-d35b1680-623d-11e9-92fc-29f1aa5c42ff.png">

**Comment by siman** [comment]
Right, it doesn't work in our current UI, but I also tested this code snipped in UI that is merged w/ Polkadot upstream and it works. So it should work if we merge this PR https://github.com/Joystream/apps/pull/137

**Comment by bwhm** [comment]
Great work guys :)

**Comment by mnaamani** [comment]
Fixed in #137
